pkg/converters/ingress/ingress.go

```diff
@@ -597,6 +611,27 @@ func (c *converter) addBackend(source *annotations.Source, hostname, uri, fullSv
 	return backend, nil
 }
 
+func (c *converter) syncBackendEndpointCookies(backend *hatypes.Backend) {
+	cookieAffinity := backend.CookieAffinity()
+	for _, ep := range backend.Endpoints {
+		if cookieAffinity {
+			switch backend.EpCookieStrategy {
+			default:
+				ep.CookieValue = ep.Name
+			case hatypes.EpCookiePodUid:
+				if ep.TargetRef != "" {
+					pod, err := c.cache.GetPod(ep.TargetRef)
+					if err == nil {
+						ep.CookieValue = fmt.Sprintf("%v", pod.UID)
+					} else {
+						c.logger.Error("error calculating cookie value for pod %s: %v", ep.TargetRef, err)
```

Comment:
`backend.AcquireEndpoint()` doesn't fill CookieValue, shouldn't be assigned to ep.Name in the case of an error reading the pod UID?

Reply:
Implemented in https://github.com/jcmoraisjr/haproxy-ingress/pull/667/commits/0d58edb40287f5f1e19770498566e8776fb5cd50